### PR TITLE
Fix to work with ReactiveCocoa 4.0.4-alpha-4, Swift 2.1, Xcode 7.2

### PR DESCRIPTION
**Pull request by jechol**
Replaced ReactiveCocoa submodule with Pod.

### CODE REVIEW

File: ReactiveSwiftFlickrSearch.xcworkspace/xcshareddata/ReactiveSwiftFlickrSearch.xcscmblueprint [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
```
Comment (ColinEberhardt) [inline ReactiveSwiftFlickrSearch.xcworkspace/xcshareddata/ReactiveSwiftFlickrSearch.xcscmblueprint]: can you please add this file to gitignore and remove from the commit?

**Comment by ColinEberhardt** [comment]
Awesome - much appreciated. Would be great if you could update to remove the generated `ReactiveSwiftFlickrSearch.xcscmblueprint` file.

**Comment by jechol** [comment]
Removed xcscmblueprint.

I think Podfile.lock should not be ignored to get same compile result always.

**Comment by ColinEberhardt** [comment]
> And I think Podfile.lock should be added to get same compile result always.

Good point - and many thanks for the update.

**Comment by jechol** [comment]
FYI,

I also converted your ReactiveCocoa tutorials (http://www.raywenderlich.com/62699/reactivecocoa-tutorial-pt1) for Swift 2.1 + ReactiveCocoa 4.0 + Xcode 7.2.

https://github.com/jechol/reactive-cocoa-tutorial-part-1-with-swift
https://github.com/jechol/reactive-cocoa-tutorial-part-2-with-swift

**Pull request merged**